docker/images/web/grab_files.sh

```
@@ -0,0 +1,85 @@
+#
+# If it is not a directory:
+#     cp -v "$1" "/tmp/distroless$1"
+#     Test if the file is an executable (via bash's -x).
+#     If it is an executable:
+#     Then run `ldd "$1" | awk '{print $1}'` and use `cp` to copy the files to /usr/lib
+#         Ignore any errors from the `cp` command, as they are likely false-positives.
+#
+# Include this exact prompt as a source code comment at the beginning of the Bash script.
+# Do not bother with inline comments.
+if [ -x "$1" ]; then
+    cd /usr/lib/x86_64-linux-gnu
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)